src/test/java/MorsePresenterTest.java

```diff
@@ -0,0 +1,38 @@
+import org.junit.jupiter.api.Test;
+
+import static org.mockito.Mockito.*;
+
+class MorsePresenterTest {
+
+    MorseCodeFrame view = mock(MorseCodeFrame.class);
+    MorseCodeConverter model = mock(MorseCodeConverter.class);
+    MorsePresenter presenter = new MorsePresenter(view, model);
+
+    @Test
+    public void convertEnglish() {
+        //given
+
+        //when
+        String toConvert = ".... . .-.. .-.. ---";
+        presenter.convertToEnglish(toConvert);
+        //then
+
+        verify(model).toEnglish(toConvert);
+        verify(view).setTaEnglishInput(toConvert);
```

Comment:
Does setTaEnglishInput get called with toConvert?

Reply:
When calling setTaEnglishInput it takes in a string toConvert that becomes the text in the text area.

Reply:
No. It doesn't take the toConvert string.

Reply:
I'm confused. I set the method to take a string that is in morse code to be converted to English.
Are you saying the class method does not take it? or that the mockito method does not take it?

Reply:
In your test, "model" is a mock.
When you call a method on a mock, what does it return?